server/src/main/java/org/candlepin/model/CertificateSerial.java

```diff
@@ -49,20 +49,15 @@
     /*
      * A CertificateSerial is considered revoked when no certificates reference it
      * TODO: put different kinds of serials into different tables.  More specifically,
-     * those that we own (can revoke) vs those that we don't
+     * those that we own (can revoke) vs those that we don't.
```

Comment:
The above comment about splitting tables can likely be removed given the changes made here.